firewall.py

```diff
@@ -0,0 +1,97 @@
+import logging
+
+class Firewall:
```

Comment:
this class is just called "Firewall", but it seems to hard code the firewall configuration for a particular use case.